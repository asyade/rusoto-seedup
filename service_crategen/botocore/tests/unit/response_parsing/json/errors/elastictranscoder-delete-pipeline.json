{
  "message": "1 validation error detected: Value 'foobar' at 'id' failed to satisfy constraint: Member must satisfy regular expression pattern: ^\\d{13}-\\w{6}$",
  "__headers__": {
    "x-amzn-requestid": "1234",
    "content-length": "120",
    "date": "Wed, 19 Feb 2014 22:13:13 GMT",
    "nncoection": "close",
    "content-type": "application/json",
    "x-amzn-errortype": "ValidationException:"
  }
}
