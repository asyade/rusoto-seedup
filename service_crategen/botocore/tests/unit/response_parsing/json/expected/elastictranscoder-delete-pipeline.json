{
    "ResponseMetadata": {"RequestId": "1234"},
    "Error": {
        "Message": "1 validation error detected: Value 'foobar' at 'id' failed to satisfy constraint: Member must satisfy regular expression pattern: ^\\d{13}-\\w{6}$",
        "Code": "ValidationException"
    }
}
